inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-// Copyright 2015
-// Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
-// Technische Universität Darmstadt
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-// http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+Getting Started
 
-= Getting started
+This Getting-Started-Guide shall assist new users to install and start
+and work with INCEpTION. It gives a quick overview (estimated time for
+reading only: approx. 15 minutes) on the main issues in order to get
+familiar with the tool and does not incude special cases and details due
+to simplicity and focus on the first steps. For further reading on any
+issue, see our main documentation.: LINK EINFÜGEN. For quick overviews,
+also see our tutorial videos (Topics: Introduction, Overview,
+Recommender Basics, Entity Linking):
+https://www.youtube.com/playlist?list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-
 
-In order to run {product-name}, you need to have Java installed on your system in version 8 or 
-higher. If you do not have Java installed yet, please the latest link:https://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java] or link:https://adoptopenjdk.net[OpenJDK].
+ +
+_Introduction_
 
-Download the stand-alone JAR from the link:https://inception-project.github.io/downloads/[{product-name} downloads page]. 
+* What can INCEpTION be used for? (also see our introduction video for a
+first impression what INCEpTION is):
+https://www.youtube.com/watch?list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&time_continue=135&v=Ely8eBKqiSI&feature=emb_logo)
 
-Start the application simply by *double-clicking* on the download JAR file in your file manager.
-After a moment, a splash screen will be displayed while the application is initializing. Once the
-initialization is complete, a dialog will appear which you can use to open the application in your
-default browser or to shut down the application.
+* {blank}
+** INCEpTION is a text-annotation enviroment useful for various kinds of
+annotation projects. The core functionality is annotating any kind of
+written text. Annotations can be used both for linguistic and machine
+learning concerns.
+** But INCEpTION is not only an annotation tool. It integrates three
```

Comment:
Remove `but`

Reply:
What do you not like about it?
Without "but" or the like, the connection is missing. => I replaced "But" by "What's more on... is that it..." (though I find "But there is more..." more clear since we have a contrast here). Are you ok with that?

Reply:
But has a negative connotation to me. Also, when I read "not only", I expect an also